src/components/WalletModal/OptionsView.vue

```diff
@@ -4,7 +4,7 @@
                  :header="optionItem.name"
                  :key="key"
                  :icon="require('./../../assets/icons/' + optionItem.iconName)"
-                 :active="connector && connector.name === optionItem.connector.name"
+                 :active="connector && connector.name === optionItem.connector.name && (connector.name !== 'BlockWallet' || connector.name === optionItem.name)"
```

Comment:
Thanks very much for this PR @ImanH , I just have one more change to request, I think just this change will make this PR ready for merge (basically when connecting with MetaMask it would show both BlockWallet and MetaMask as active, but yeah at the moment the active dot is working when connecting with BlockWallet but not with MetaMask), this corrects it:

`:active="connector && connector.name === optionItem.name"`

Reply:
@JayWelsh thanks a lot! Sorry, I missed that case. I now applied your suggested change :)